source/MaterialXFormat/GraphIO.h

```diff
@@ -0,0 +1,436 @@
+//
+// TM & (c) 2022 Lucasfilm Entertainment Company Ltd. and Lucasfilm Ltd.
+// All rights reserved.  See LICENSE.txt for license.
+//
+
+#ifndef MATERIALX_GRAPHIO_H
+#define MATERIALX_GRAPHIO_H
+
+/// @file
+/// Support for the MaterialX GraphElement interchange classes
+
+#include <MaterialXCore/Library.h>
```

Comment:
I worked through the mental exercise of how generic this file is ~ i.e. could I use it to traverse a non-MaterialX graph? As a result I'm curious about the need to include anything but Export.h here? I'm not spotting the use of anything MaterialX specific in this header, aside from the namespace and API macros.